qdrant-landing/content/documentation/cloud/backups.md

```diff
@@ -1,36 +1,95 @@
 ---
 title: Backups
-weight: 30
+weight: 60
 ---
 
-# Backups
+# Cloud Backups
 
-There are situations where you need to restore your cluster because of application or system failure.
-In most cases you will have a source of truth for your data in a regular database and would be able to reindex the data into your Qdrant vector search cluster.
-However, encoding and uploading a big amount of data might require a long time.
-For high availability critical projects we highly recommend relying on replication, which is always a better option, because it guarantees the proper cluster functionality as long as at least one replica is running.
-For less critical use-cases you can make use of one of the available options.
+Qdrant organizes cloud instances as clusters. On occasion, you may need to
+restore your cluster because of application or system failure.
+
+You may already have a source of truth for your data in a regular database. If you
```

Comment:
The line break should not be mid sentence.

Reply:
There are similar cases below.

Reply:
The line break location does not matter in Markdown. Static site generators process a line break as if it were a space between words.

The advantage: if someone needs to suggest a change, it's a lot easier to read.

So in my past work as a Technical Writer, I've learned (and have been told) to limit my lines to 80 characters.

Reply:
The standard Markdown linter default has a [line limit of 80 characters](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md013---line-length) (with exceptions)